bitnami/geode/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+annotations:
+  category: Database
+apiVersion: v2
+appVersion: 1.14.0
+dependencies:
```

Comment:
Could you describe the reasons why not to use the `haproxy` chart as dependency, instead of including the container only?

Reply:
It's mandatory that the `haproxy` container **is launched in the same pod** as Apache Geode in order to access the metrics that are **only accessible via localhost**. Using the chart does not help solving the issue since `haproxy` containers will be launched on different pods.

Reply:
Makes sense!